### PR TITLE
create personal access token link was broken

### LINKED ISSUE #21173: Create personal access token link is broken.
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

Sir/Ma'am  kindly assign this task to me.

### What part(s) of the article would you like to see updated?

Create personal access token. 

### Additional information

Link is broken.

### PR DESCRIPTION
**Pull request by dhruv-porwal**
i found that create  personal access token link was broken so, added correct  github docs link to it for use.

### Why:

Closes https://github.com/github/docs/issues/21173

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
Screenshot_20221007-142805_Chrome
Screenshot_20221007-142805_Chrome
Screenshot_20221007-143011_Chrome

### CODE REVIEW

**Comment by dhruv-porwal** [comment]
Sir kindly review it.

**Comment by dhruv-porwal** [comment]
sir/ma'am merge it please

**Comment by dhruv-porwal** [comment]
Sir add it now.

File: content/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on.md [hunk 1]
```diff
@@ -29,5 +29,5 @@ You can authorize an existing personal access token, or [create a new personal a
 
 ## Further reading
 
-- "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)"
+- "[Creating a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)"
```
Comment (subatoi) [inline content/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on.md:32]:
```suggestion
- "[Creating a personal access token](/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)"
```

We use relative URLs conventionally

**Comment by subatoi** [comment]
Thanks for your contribution, @dhruv-porwal! I'll get this one merged for you.

**Comment by dhruv-porwal** [comment]
> Thanks for your contribution, @dhruv-porwal! I'll get this one merged for you.

Thank you Sir

**Review by subatoi — APPROVED** [review]

**Comment by dhruv-porwal** [comment]
sir kindly mark it hacktoberfest  accepted

**Comment by dhruv-porwal** [comment]
please  sir

**Comment by DarkDose75** [comment]
image

**Comment by DarkDose75** [comment]
Done

**Pull request merged**